<launch>
    <!--PyTorch-->
    <param name="net" type="str" value="pytorch" />
    <param name="arch" type="str" value="ssdlite320_mobilenet_v3_large" />
    <param name="device" type="str" value="cpu" />
    <param name="config" type="str" value="$(find pytorch)/models/ssdlite320_mobilenet_v3_large_coco-a79551df.pth" />
    <param name="model" type="str" value="$(find pytorch)/models/ssdlite.py" />
    <param name="image_size" type="int" value="224" />
    <param name="classes" type="str" value="$(find pytorch)/models/frame_based/action_classes.txt" />
    <!--param name="classes" type="str" value="$(find pytorch)/models/model_pytorch/object_detection_classes_coco.txt" /-->
    <param name="threshold" type="double" value="0.4" />
    <param name="input_topic" type="str" value="/usb_cam/image_raw" />
    <param name="detection_topic" type="str" value="/image_detection" />

    <node name="object_detector" pkg="pytorch" type="detect.py" output="screen"/>
</launch>